html/user/get_project_config.php

```diff
@@ -66,10 +66,6 @@ function show_platforms() {
     <web_rpc_url_base>".secure_url_base()."</web_rpc_url_base>
 ";
 
-if ( isset($git_commit) ) {
```

Comment:
Does this mean that the project config should not communicate any codebase details anymore? Why not replace this with the newly generated common tag base?

Reply:
No it now only uses the tag.

Reply:
Ah, ok, I [found it](https://github.com/BOINC/boinc/blob/57e2e5b66dfd588575812638d13f01b49c29de25/html/user/get_project_config.php#L69) myself.